README.md

```diff
@@ -1,23 +1,17 @@
 # ImageHasher
 
+An executor to encode the images into embeddings using different comparable hashing technique, such as `perceptual`,
+`average`, `wavelet`, and `difference` techniques. Features in the image are used to generate a distinct
+(but not unique) fingerprint, and these fingerprints are comparable.
 
-## Usage
+Comparable hashes are a different concept compared to cryptographic hash functions like `MD5` and `SHA1`. With
```

Comment:
The formatting of the endlines is odd. Not a big deal

Reply:
Used PyCharm to auto format the file